scripts/west_commands/blobs.py

```diff
@@ -71,11 +71,12 @@ def do_add_parser(self, parser_adder):
                             help='''Format string to use to list each blob;
                                     see FORMAT STRINGS below.''')
 
-        parser.add_argument('-m', '--modules', type=str, action='append',
+        parser.add_argument('-m', '--module', type=str, action='append',
                             default=[],
-                            help='''a list of modules; only blobs whose
+                            help='''a module name; only blobs whose
                             names are on this list will be taken into account
-                            by the sub-command. Invoke multiple times''')
+                            by the sub-command. Invoke multiple times to specify
+                            multiple modules.''')
```

Comment:
shouldn't `-m` be removed as to be consistent with `west update` (and `git fetch`) where you don't need extra args when on wanting a specific module.

```
usage: west update ... [PROJECT [PROJECT ...]]

```

Reply:
Fair enough. For some reason this did not work earlier when combined with other flags, but it now does, so I've reverted to this behavior. Done.